src/games/strategy/engine/data/export/RulesAttachmentExporter.java

```diff
@@ -134,11 +135,9 @@ private String mTurnsHandler(final Field field, final IAttachment attachment) {
     try {
       oValue = field.get(attachment);
     } catch (final IllegalArgumentException e) {
```

Comment:
Side comment - we should not be catching IllegalArgumentException. It's unchecked, and for the most part the programmer should ensure they do not call a method with bad args. If we just rely on the exception coming back when there are bad or missing args, that is called "control flow by exception handling". #endrant